cnxepub/models.py

```diff
@@ -83,9 +82,9 @@ def utf8(item):
         return item
 
 
-def _sanitize_xml(raw_xml):
+def _sanitize_xml(raw_xml, recover=True):
```

Comment:
Is there any reason we should use `recover=True`? It seems that by keeping it `recover=True` we are leaving this little landmine/rabbit-hole for future content... right?

Reply:
I think we should check what's going to break, instead of just changing it.  I think it can be raised as a separate issue.  I have added this to Connexions/cnx-archive#518 for error handling.